src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -0,0 +1,314 @@
+        maxFeePerGas: maxFeePerGasMedium.value,
+      },
+      {
+        fee: feeHigh.value,
+        time: 25,
+        label: t('common.transferSpeed.fast'),
+        maxPriorityFee: maxPriorityFeePerGasFast.value,
+        maxFeePerGas: maxFeePerGasHigh.value,
+      },
+    ]);
+
+    const fee = computed(() => feeList.value[feeSelectedIndex.value].fee);
+
+    const numericFee = computed(() => +fee.value.toFixed());
+    const max = computed(() => balance.value.minus(fee.value));
+
+    function emitCurrentFormModelState() {
+      const inputPayload: TransferFormModel = {
+        ...formModel.value,
+        fee: fee.value as BigNumber,
+        total: numericFee.value + +(formModel.value?.amount || 0),
+        invoiceId: invoiceId.value,
+        invoiceContract: invoiceContract.value,
+      };
+      emit('update:transferData', inputPayload);
+      return nextTick();
+    }
+
+    // Method called from a parent scope - avoid changing its name.
+    async function submit() {
+      if (!hasError.value) {
+        await emitCurrentFormModelState();
+        emit('success');
+      }
+    }
+
+    function setMaxAmount() {
+      formModel.value.amount = max.value.isPositive() ? max.value.toString() : '0';
+    }
+
+    async function updateFeeList() {
+      const baseFee = new BigNumber(await getEthBaseFee());
+
+      maxFeePerGasSlow.value = baseFee.multipliedBy(2).plus(maxPriorityFeePerGasSlow.value);
+      maxFeePerGasMedium.value = baseFee.multipliedBy(2).plus(maxPriorityFeePerGasMedium.value);
+      maxFeePerGasHigh.value = baseFee.multipliedBy(2).plus(maxPriorityFeePerGasFast.value);
+
+      feeSlow.value = baseFee.plus(maxPriorityFeePerGasSlow.value);
+      feeMedium.value = baseFee.plus(maxPriorityFeePerGasMedium.value);
+      feeHigh.value = baseFee.plus(maxPriorityFeePerGasFast.value);
+    }
+
+    let polling: NodeJS.Timer | null = null;
+
+    onMounted(() => {
+      polling = executeAndSetInterval(() => {
+        updateFeeList();
+      }, 5000);
+
+      const { query } = route;
+      updateFormModelValues({
+        ...query,
+        token: query.token,
+      });
+    });
+
+    onUnmounted(() => {
+      if (polling) {
+        clearInterval(polling);
+      }
+    });
+
+    watch(
+      hasError,
+      (val) => emit('error', val),
+      { deep: true },
+    );
+
+    watch(
+      formModel,
+      () => {
+        emitCurrentFormModelState();
+      },
+      { deep: true },
+    );
+
+    return {
+      ETH_COIN_NAME,
+      ETH_SYMBOL,
+      PROTOCOL_ETHEREUM,
+      NETWORK_TYPE_TESTNET,
+      formModel,
+      activeNetwork,
+      fee,
+      feeList,
+      feeSelectedIndex,
+      numericFee,
+      errors,
+      balance,
+      max,
+      openScanQrModal,
+      handleAssetChange,
+      submit,
+      setMaxAmount,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+
+.transfer-send-form {
+  .max-button {
```

Comment:
This is the 4th place in the app that the same styles are used. I suggest to create separate button component that would use the `BtnPlain` as it's base.